physionet-django/events/views.py

```diff
@@ -310,3 +314,46 @@ def event_detail(request, event_slug):
          'is_waitlisted': is_waitlisted,
          'event_datasets': event_datasets,
          })
+
+
+@require_http_methods(["PUT", "DELETE", "GET", "POST"])
+def manage_co_hosts(request, event_slug):
+    """
+    Manage co-hosts of an event
+    """
+    user = request.user
+    error_message = None
+
+    body = json.loads(request.body)
+    participant_id = body.get('participant_id')
```

Comment:
Now that I think of it, a `DELETE` call shouldn't contain a body either (it's semantics are undefined). Maybe this endpoint should be redesigned after all.